### DISCUSSION TITLE
Can you change the weather widget from fahrenheit to celcius?

### DISCUSSION THREAD

**Discussion by imtsubi** [MM2-0/Kvaesitso#648] [Q&A] [answered]
I recently got the Kvaesitso launcher and have been having trouble finding an option to change it to celcius, I'm not even sure if you can but i still tried to look. Does anyone know if you can, if so, how?

**Comment by MM2-0** [comment, marked as answer]
Settings > integrations > weather

**Reply by imtsubi** [reply]
Thank you so much!

**Reply by kartik2005221** [reply]
can u please provide weather from more countries, i am in india, haryana, kanina khas 123027...please
